# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/mozilla/tests/focus/focus-next-tick-before-iframe-loaded-different-site.html
# Commit: c63ecdad6a3c
# Full Hash: c63ecdad6a3c60ba625517d34bb29c8aae9f5e95
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-02-22 17:34:52
# Regressor Bug: 1663931
# File Overlap Count: 1
# Description:
#   Bug 1663931 - Avoid moving focus when changing iframe remoteness. r=nika,mccr8
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D99215
# ==============================================================================

diff -r 828c62bfc0a2 -r c63ecdad6a3c testing/web-platform/mozilla/tests/focus/focus-next-tick-before-iframe-loaded-different-site.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/mozilla/tests/focus/focus-next-tick-before-iframe-loaded-different-site.html	Mon Feb 22 10:51:51 2021 +0000
@@ -0,0 +1,16 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>focus() from next tick before iframe loaded different site</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+setup({explicit_done:true});
+window.onmessage = function(e) {
+    test(function() {
+        assert_equals(e.data, "PASS", "Check verdict");
+    }, "Check result");
+    w.close();
+    done();
+};
+var w = window.open("support/focus-next-tick-before-iframe-loaded-different-site-outer.sub.html");
+</script>